src/modeler/AutoRest.Swagger/Validation/Extensions/LongRunningResponseValidation.cs

```diff
@@ -26,6 +36,6 @@ public class LongRunningResponseValidation : LongRunningExtensionRule
         /// <summary>
         /// The severity of this message (ie, debug/info/warning/error/fatal, etc)
         /// </summary>
-        public override Category Severity => Category.Warning;
+        public override Category Severity => Category.Error;
```

Comment:
Nit: Would it make sense to run some of these rules against the rest specs repo before upgrading the severity status